game/scripts/npc/items/item_siege_mode.txt

```diff
@@ -92,7 +92,7 @@
       "06"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "siege_atkspd_bonus"                              "0"
```

Comment:
Well I assumed the value was going to be turned negative. It's a little confusing how it's worded.